Karnayxov

AndreyKarn opened the pull request.

AndreyKarn committed:
Контрольная работа 1

КР 1
Карнаухов А. А.
ЭР-15-16

AndreyKarn committed:
ЛР_1

ЛР 1
Карнаухов А.А.
ЭР-15-16

Korogodin commented:
По лабораторной работе.

С технической стороны всё ок, надо подтянут оформление к ГОСТ 7-32:
1. Отступ красной строки 1.25
2. Подписи рисунков вида: Рисунок ХХ - Подпись
3. После нумерации и в конце заголовка точка не ставится

Korogodin commented:
По контрольной, не понял описание функции B(t):
![image](https://user-images.githubusercontent.com/747285/96222522-1910b080-0f95-11eb-970d-63ceba20d6ff.png)

Как выглядит B(t)?

AndreyKarn committed:
ЛР 1 с исправлениями

Высылаю исправленную версию отчета по ЛР 1. А насчет КР 1, то вы ее уже проверили, я просто случайно загрузил ее исправленную версию не в свой репозиторий, а в Ваш, а как перенести я не знаю.

AndreyKarn committed:
Delete Карнаухов_ЛР_1_испр.pdf

AndreyKarn committed:
ЛР_1_испр

AndreyKarn commented:
Загрузил исправленную ЛР1, а контрольную я исправил и Вы ее проверили, просто я случайно загрузил файл в Ваш репозиторий, а как перенести я не знаю

Korogodin commented:
ЛР1 принята
